### DISCUSSION TITLE
Can't get Whisper to use my GPU

### DISCUSSION THREAD

**Discussion by polymood** [openai/whisper#1173] [Q&A] [answered]
I can't get Whisper to use my GPU (RTX 2060), and going through the discussions I can't fixe the issue.

Is there a specific way to use GPUs, and codewise with python?

**Comment by jongwook** [comment, marked as answer]
You'll need the CUDA drivers installed along with the PyTorch package built for GPU. You can follow [this page](https://pytorch.org/get-started/locally/) to install PyTorch locally, and get more help on [PyTorch Forums](https://discuss.pytorch.org/).
